upcase_booklist_app/templates/book.html

```diff
@@ -16,21 +16,19 @@
         </nav>
 
         <div>
-            <div class="single-book">
-
-                <!-- One parent -->
-                <div class="single-book-image"><img src="{{ book['image'] }}" alt="{{ book['title'] }}"></div>
-            
-                <!-- Second parent with book details -->
-                <div class="display-single-book">
-                    <div class="space single-book-title">{{ book['title'] }}</div>
-                    <div class="space single-book-author"><br>By: {{ book['author'] }}</div>
-                    <div class="space single-book-summary"><br>{{ book['summary'] }}</div>
+                <div class="single-book">
+    
+                    <!-- One parent -->
+                    <div class="single-book-image"><img src="{{ book['image'] }}" alt="{{ book['title'] }}"></div>
```

Comment:
Excellent use of the `book` variable that contains all the dynamic data for this template.